app/code/Magento/Config/Model/Config/Backend/File.php

```diff
@@ -82,12 +84,14 @@ public function __construct(
      * Save uploaded file before saving config value
      *
      * @return $this
-     * @throws \Magento\Framework\Exception\LocalizedException
+     * @throws LocalizedException
+     * @SuppressWarnings(PHPMD.CyclomaticComplexity)
```

Comment:
Please try to refactor to avoid adding `PHPMD.CyclomaticComplexity`

Reply:
there is no CyclomaticComplexity anymore after the fix below